contrib/tumbler/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "tumbler"
+pkgver = "4.18.2"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gettext",
```

Comment:
this doesn't build without this being gettext-devel

Reply:
so far also found in xfce4-settings, possibly others

Reply:
strange, i build-tested literally everything but okay